spaceros/README.md

```diff
@@ -78,12 +80,21 @@ Commands:
   Call `ros2 <command> -h` for more detailed usage.
 ```
 
-## Running the Space ROS Unit Tests
+## Rebuilding Space ROS and running unit tests
+
+Space ROS sources and intermediate build artifacts are not included in the docker image.
 
-The Space ROS unit tests are not built as part of the Docker image build. To run the unit tests in the container, execute the following command from the top of the Space ROS workspace:
+A manifest of the exact sources used to produce the current image is saved as `exact.repos` in the spaceros directory.
+To clone all sources from this manifest you can use the command sequence
+```
+spaceros-user@d10d85c68f0e:~/spaceros$ mkdir src
+spaceros-user@d10d85c68f0e:~/spaceros$ vcs import src < exact.repos
+```
 
+From there you can run a new build and any additional tests.
 ```
-spaceros-user@d10d85c68f0e:~/spaceros$ colcon test
+spaceros-user@d10d85c68f0e:~/spaceros$ colcon build --cmake-args -DCMAKE_BUILD_TYPE=Debug --DCMAKE_EXPORT_COMPILE_COMMANDS=ON --no-warn-unused-cli
+spaceros-user@d10d85c68f0e:~/spaceros$ colcon test --ctest-args -LE "(ikos|xfail)" --pytest-args -m "not xfail"
```

Comment:
@mjeronimo this whole section may be a bit of an overstep for this README. I think moving it to the docs site either in https://github.com/space-ros/docs/pull/5 or another tutorial specifically about the docker image may make sense.

If we do want to leave it here, I updated the `colcon test` command to match the args used during automated builds (minus retests).

Reply:
We can work out the proper split when we update the documentation site. I've been thinking about this topic and what information goes where. Good to have it here for now and we can refactor.